tests/IsoCodes/Tests/AbstractIsoCodeTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace IsoCodes\Tests;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+abstract class AbstractIsoCodeTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @return array[]
+     */
+    abstract public function getValidValues();
+
+    /**
+     * @return array[]
+     */
+    abstract public function getInvalidValues();
+
+    final public function testEmptyValues()
```

Comment:
The goal of this method is to be sure to test empty values on each validator without duplicate provided data on each validator test class.